docs/production/install.md

```diff
@@ -119,6 +119,12 @@ of the failure, you can just rerun the script. For more information, see
   notifications are not enabled, no data will be submitted, so this flag is
   redundant.
 
+- `--agree-to-terms-of-service`: If you're using the `--push-notifications` flag,
+  you can pass this additional flag to indicate that you have read and agree to
+  the [Terms of Service][terms].
+  This skips the Terms of Service prompt, allowing for running the installer
+  with `--push-notifications` in scripts without requiring user input.
```

Comment:
We may want to move this to the advanced installer options section, just to keep this page really short. I think I'll merge this and we can discuss that potential polish opportunity.